app/_hub/kong-inc/kafka-log/index.md

```diff
@@ -32,87 +33,103 @@ params:
       value_in_examples: BOOTSTRAP_SERVERS
       urlencode_in_examples: true
       default:
+      datatype: set
       description: |
         List of bootstrap brokers in a `{host: host, port: port}` format.
```

Comment:
```suggestion
        Set of bootstrap brokers in a `{host: host, port: port}` list format. Refer to the "Quickstart" section for examples.
```

we already have an `application/x-www-form-urlencoded` example in the "Quickstart" section. I'd suggest adding an `application/json` example there as well. Here is my example JSON config:

```json
{
  "name": "kafka-log",
  "config": {
    "topic": "kong-log",
    "bootstrap_servers": [
      {
        "host": "localhost",
        "port": 9092
      }
    ]
  }
}
```

Reply:
@murillopaula thanks for the extra example. We should avoid using double quotes and I'll make links between the sections. Also, I still think the description is wrong? Should be host:port, host:port  and not host:host, port:port?

Reply:
@murillopaula since the entire doc is currently `urlencode_in_examples: true`, and the main focus of this PR is to add datatypes, I'll log a ticket for us to work on an entire json example rather than just an inserted snippet.